.kitchen.vagrant.yml

```diff
@@ -1,44 +0,0 @@
----
-driver:
```

Comment:
@chris-rock I have the same view more or less. I will have a look how to rework it with kitchen local approach, its a good idea, thanks!

> I agree that we need to have another solution for os-hardening since we are testing kernel parameters, but I am not sure we should switch all other projects.

I'm not sure as well, but lets have this discussion outside of this PR